src/index.html

```diff
@@ -394,6 +400,25 @@ <h2 class="font-weight-bold">Contact</h2>
                 </div>
             </form>
         </section>
+
+        <section class="col-12 py-5" id="board-members">
```

Comment:
@alevi22 and @ramesh-arjun, does this section make more sense after the contact section or after the sponsor section? 🤔

Reply:
@LeahOlson I would put it after the sponsors and partners section!